### DISCUSSION TITLE
横幅が1200pxちょうどの時にレイアウトが崩れる

### DISCUSSION THREAD

**Discussion by semimaru141** [increments/qiita-discussions#506] [要望・アイデア・不具合]
# 概要
横幅を1200pxちょうどにした際にQiitaのボタン部分などのレイアウトが崩れ、以下のように表示されてしまっている。
（いいねボタンなどが画面外にはみ出てしまっている）

![スクリーンショット 2023-07-27 16 23 05](https://github.com/increments/qiita-discussions/assets/45461194/1878dd47-d853-4b28-b972-63edd0af641b)

# 確認環境
MacBookPro M2 13インチ
macOS Ventura バージョン 13.3.1
Google Chrome バージョン: 112.0.5615.137（Official Build） （arm64）
動作確認日: 2023/07/27

# 再現方法と思われるもの
全てのサイトに対して発生するわけではないようです。
自分が確認したURLを以下に貼っておきます。
https://qiita.com/nrslib/items/a5f902c4defc83bd46b8
https://qiita.com/shizen-shin/items/10f8b0d5cccbbaf139bf
https://qiita.com/ishiyama0530/items/bffaeca502507dc98262
https://qiita.com/P-man_Brown/items/013366d5e1f34353c575

**Comment by wataru86** [comment]
@semimaru141

ご報告ありがとうございます 🙏
現在、修正対応を進めています！
修正が完了したらまたご連絡いたします！

**Comment by wataru86** [comment]
@semimaru141

お待たせしました🙇
こちら修正が完了いたしました！

**Reply by semimaru141** [reply]
対応お疲れ様です！
自分の提示しました４つの記事でレイアウトが崩れないことを確認しました！

このissueを解決として、ディスカッションをクローズします！

**Comment by semimaru141** [comment]
解決によりクローズ
